p2p/src/test/kotlin/maru/p2p/P2PTest.kt

```diff
@@ -1157,8 +1157,8 @@ class P2PTest {
     peer: String,
   ) {
     assertThat(
-      p2pNetwork.isConnected(peer),
-    ).isTrue()
+      p2pNetwork.getPeerLookup().getPeer(LibP2PNodeId(PeerId.fromBase58(peer)))
```

Comment:
Now it doesn't verify what the name of the function claims

Reply:
You mean that if a peer is found in the lookup, it means it's connected?

Reply:
Yes. Peer lookup uses the connectedPeers from the MaruPeerManager, which contains the peers that we are connected to.